### PR TITLE
SNS auto-confirmation, which is needed by terraform

### PR DESCRIPTION
**Pull request by igorwwwwwwwwwwwwwwwwwwww**

### CODE REVIEW

File: workers/sns_messages.go [hunk 1]
```diff
@@ -54,6 +54,39 @@ func snsMessagesMain(cfg *internalConfig, msg *workers.Msg) {
 	}
 }
 
+// http://docs.aws.amazon.com/sns/latest/dg/SendMessageToHttp.html
+func handleSNSConfirmation(rc redis.Conn, msg *pudding.SNSMessage) error {
+	if os.Getenv("SNS_CONFIRMATION") == "1" || os.Getenv("SNS_CONFIRMATION") == "true" {
```
Comment (meatballhat) [inline workers/sns_messages.go]: nitpick: `if v, _ := strconv.ParseBool(os.Getenv("SNS_CONFIRMATION")); v {`

**Comment by meatballhat** [comment]
:shipit:

File: workers/sns_messages.go (hunk 1 above)
Comment (igorwwwwwwwwwwwwwwwwwwww) [inline workers/sns_messages.go]: TIL. Very nice! :)

**Comment by meatballhat** [comment]
:shipit: 🐑 :it:

**Comment by igorwwwwwwwwwwwwwwwwwwww** [comment]
works <3

**Pull request merged**